src/ResourceManagement/Sql/SqlChildrenOperationsImpl.cs

```diff
@@ -0,0 +1,132 @@
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Management.Sql.Fluent.SqlChildrenOperations.SqlChildrenActionsDefinition;
+    using System.Collections.Generic;
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+
+    /// <summary>
+    /// Implementation for SQL Server children operations.
+    /// </summary>
+    /// <typeparam name="FluentModelT">The fluent model type of the child resource.</typeparam>
+    ///GENTHASH:Y29tLm1pY3Jvc29mdC5henVyZS5tYW5hZ2VtZW50LnNxbC5pbXBsZW1lbnRhdGlvbi5TcWxDaGlsZHJlbk9wZXJhdGlvbnNJbXBs
+    internal abstract partial class SqlChildrenOperationsImpl<FluentModelT>  :
+        ISqlChildrenOperations<FluentModelT>,
+        ISqlChildrenActionsDefinition<FluentModelT>
+    {
+        protected ISqlManager sqlServerManager;
+        protected ISqlServer sqlServer;
+
+        ///GENMHASH:38F25BF3BAF6CC66FCF20A982BCC9D49:D2D2D6E5CB31EA693A417EE1C7AA57DF
+        internal SqlChildrenOperationsImpl(ISqlServer parent, ISqlManager sqlServerManager)
+        {
+            this.sqlServer = parent;
+            this.sqlServerManager = sqlServerManager;
+        }
+
+        ///GENMHASH:5002116800CBAC02BBC1B4BF62BC4942:DFB9811D4C0F33590C78A6AF9E912994
+        public FluentModelT GetById(string id)
+        {
+            return this.GetBySqlServer(ResourceUtils.GroupFromResourceId(id),
+                ResourceUtils.NameFromResourceId(ResourceUtils.ParentRelativePathFromResourceId(id)),
+                ResourceUtils.NameFromResourceId(id));
+        }
+
+        ///GENMHASH:206E829E50B031B66F6EA9C7402231F9:98A40305CB5222ADE48FAC15EF20A8E7
+        public FluentModelT Get(string name)
+        {
+            if (this.sqlServer == null)
+            {
+                return default(FluentModelT);
+            }
+            return this.GetBySqlServer(this.sqlServer, name);
+        }
+
+        ///GENMHASH:E776888E46F8A3FC56D24DF4A74E5B74:487BF38141DB6ADDAB135D4A077E5776
+        public async Task<FluentModelT> GetByIdAsync(string id, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await this.GetBySqlServerAsync(ResourceUtils.GroupFromResourceId(id),
+                ResourceUtils.NameFromResourceId(ResourceUtils.ParentRelativePathFromResourceId(id)),
+                ResourceUtils.NameFromResourceId(id));
+        }
+
+        ///GENMHASH:CFA8F482B43AF8D63CC08E2DEC651ED3:DCEEEF4375DAE606E6DAD8A9FDADD8F9
+        public void DeleteById(string id)
+        {
+            this.DeleteBySqlServer(ResourceUtils.GroupFromResourceId(id),
+                ResourceUtils.NameFromResourceId(ResourceUtils.ParentRelativePathFromResourceId(id)),
+                ResourceUtils.NameFromResourceId(id));
+
+        }
+
+        ///GENMHASH:4D33A73A344E127F784620E76B686786:5C3E0E7E1CF165D67A5D6B44C259B469
+        public async Task DeleteByIdAsync(string id, CancellationToken cancellationToken = default(CancellationToken))
+        {
```

Comment:
Starting from this line and further down the code - please pass cancellationToken to the method async calls.

Reply:
Fixed